collect_app/src/main/res/values/theme.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
 
     <style name="AppThemeBase.Collect"/>
 
     <style name="AppThemeBase" parent="Theme.AppCompat.Light.NoActionBar">
+        <item name="android:textDirection" tools:targetApi="jelly_bean_mr1">locale</item>
```

Comment:
Have you tested under RTL languages? I saw you only add a `target api`  in API level 17, will that code work well in higher? Besides, I have already add these parameters in #2098 in `theme.xml` files. So I think you can remove this.

Reply:
Yes, of course, it works well.

> Besides, I have already added these parameters in #2098 in theme.xml files. So I think you can remove this.

yes but that pr is not merged yet and it's the last one marked as `needs testing` I haven't investigated carefully yet so I'll do.

Reply:
I remember you have reviewed that PR weeks ago. It’s welcome for a further investigation.
And I will not close #2099 immediately in case @lognaturel wants a hot patch in our next release, after all, I cannot see any downsides in your solution but it still need a detailed review and test. Do you agree with me? : )

Reply:
> I remember you have reviewed that PR weeks ago

yes indeed I took a look at it but now I have some second thoughts.

> And I will not close #2099 immediately in case @lognaturel wants a hot patch in our next release

I doubt but I agree let's let her decide.